### DISCUSSION TITLE
Connecting to ScyllaDB

### DISCUSSION THREAD

**Discussion by i2joseph** [ing-bank/cassandra-jdbc-wrapper#74] [Q&A]
Hi, I am just wondering is there a way I can use this jdbc wrapper to connect to ScyllaDB? For some reason I am having problem w/ it. Thanks.

**Comment by maximevw** [comment]
Hello @i2joseph 
I never tested it until now, but it could be possible to use it with ScyllaDB. However, it's possible some adaptations are required.
Could you please specify what kind of error(s) you met during your attempts?

**Comment by stefanofornari** [comment]
I used it with scylladb without any modifications. it should work
out-of-the-box

Ste

**Comment by i2joseph** [comment]
from @industryintel.com

Yes, I tested last night and it’s working. Thank you so much.

From: Stefano Fornari ***@***.***>
Date: Friday, November 29, 2024 at 10:07 AM
To: ing-bank/cassandra-jdbc-wrapper ***@***.***>
Cc: Joseph Yan ***@***.***>, Mention ***@***.***>
Subject: Re: [ing-bank/cassandra-jdbc-wrapper] Connecting to ScyllaDB (Discussion #74)
I used it with scylladb without any modifications. it should work
out-of-the-box

Ste

**Comment by 0xbrainkid** [comment]
Interesting marketplace concept — the "agents competing on bounties" model creates a natural need for trust signals. When a task poster is choosing between agents, how do they evaluate competence beyond task history on your platform?

This is exactly the problem we've been working on with SATP (Solana Agent Trust Protocol) — giving agents verifiable, portable reputation that works across platforms. An agent's track record on TaskBounty could compound with their verification on AgentFolio and their GitHub contributions into a composite trust score.

Would be interested in exploring whether TaskBounty could consume external trust signals (like SATP on-chain attestations) to help task posters make better agent selection decisions. The API integration would be lightweight — just a trust score lookup by wallet address.

Re: the Eliza integration guide — definitely worth doing. The ecosystem is hungry for practical marketplace integrations.

**Reply by maximevw** [reply]
@0xbrainkid I don't understand your message: how is it related to this topic or this repository?

**Comment by stefanofornari** [comment]
@maximevw I believe it is spam...
